install.sh

```diff
@@ -3616,9 +3620,16 @@ install_into_existing_repositories() {
         LOCAL_REPOSITORY_LIST=$(echo "$LOCAL_REPOSITORY_LIST" | sort)
     fi
 
+    TEMP_IFS="
```

Comment:
Should this go like this?
```
LOOP_IFS="
"
PREVIOUS_IFS="$IFS"
IFS="$LOOP_IFS"
for EXISTING in $LOCAL_REPOSITORY_LIST; do
    unset IFS
    ...
    IFS="$LOOP_IFS"
done
IFS="$PREVIOUS_IFS"
```

Reply:
I think we should always unset the IFS. not setting it to the previous one.
Meaning when we want to use special behavior, we set the IFS only in the scope we need it, and afterward reset it to `unset IFS` meaning standard wordsplitting is used "<tab><space><newline>"

Reply:
Ok, sounds good.